ReactiveUI/Xaml/WinRTAutoSuspendApplication.cs

```diff
@@ -30,10 +30,14 @@ public AutoSuspendHelper(Application app)
                 .Where(x => x.PreviousExecutionState == ApplicationExecutionState.Terminated)
                 .Select(_ => Unit.Default);
 
-            var unpausing = new[] { ApplicationExecutionState.Suspended, ApplicationExecutionState.Running, };
-            RxApp.SuspensionHost.IsUnpausing = _launched
-                .Where(x => unpausing.Contains(x.PreviousExecutionState))
-                .Select(_ => Unit.Default);
+			var fastResuming = new Subject<ApplicationExecutionState>();
```

Comment:
Can you fix the tabs? It looks like your editor is set for hard tabs. Just make another commit to fix it, no need to close and reopen the PR